packages/ketcher-react/src/script/ui/dialog/toolbox/labeledit.jsx

```diff
@@ -66,14 +67,28 @@ function deserialize(value) {
 
 function LabelEdit(props) {
   const init = { label: props.letter || serialize(props) }
-  const { formState, ...prop } = props
+  const { formState, onOk, ...prop } = props
   const { result, valid } = formState
+
+  const getButton = () => {
+    return [
+      <button
+        className={styles.okButton}
+        disabled={!valid}
+        onClick={() => onOk(result)}
+      >
+        Apply
+      </button>
+    ]
+  }
   return (
     <Dialog
       title="Label Edit"
-      className="labeledit"
       valid={() => valid}
+      withDivider={true}
       result={() => deserialize(result.label)}
+      className={styles.labelEdit}
+      buttons={getButton()}
```

Comment:
basically, Apply button is OK button
we can add following props:
 buttons: ['OK'],
 buttonsNameMap={{ OK: 'Apply' }}

in this case, no other changes are needed
You can check out TemplateDialog as a reference

Reply:
Ok thank u, now I understand how to use it.